templates/layouts/topLevel.jade

```diff
@@ -60,6 +60,10 @@ block generalTopLevelButtons
                         li.btn.btn-success
                                 a(href=meta.url + '/remove',title='Remove',style="color:white", onclick="return confirm('Are you sure you want to delete this?')")
                                     span.fa.fa-remove
+                if locals.user
```

Comment:
probably also `canEdit`, right?

Reply:
No, because you are not actually editing this object, you are copying it into your own collection.